<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMainWindowClass</class>
 <widget class="QMainWindow" name="KMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="m_prevButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>上一首</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_nextButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>下一首</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_playButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    qproperty-icon: url(:/images/play.png);
    
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="m_volumeSlider">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>22</width>
      <height>71</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="m_volumeButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    qproperty-icon: url(:/images/open.png);
    
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="m_volumeLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_albumCoverLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>280</width>
      <height>280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border-radius: 140px; /* 设置圆角半径为50% */
    
    border: 2px solid black;
    background-image: url(:/images/disk.png);
    background-position: center;
    background-repeat: no-repeat;
    background-size: cover;
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="m_timeSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="m_currentTimeLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_totalTimeLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_sequentialButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:checked {
    background-color: lightblue; /* 设置高亮时的背景颜色 */
    border: 2px solid #ADD8E6;
}</string>
    </property>
    <property name="text">
     <string>顺序播放</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="m_randomButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:checked {
    background-color: lightblue; /* 设置高亮时的背景颜色 */
    border: 2px solid #ADD8E6;
}</string>
    </property>
    <property name="text">
     <string>随机播放</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="m_loopOneButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:checked {
    background-color: lightblue; /* 设置高亮时的背景颜色 */
    border: 2px solid #ADD8E6;
}</string>
    </property>
    <property name="text">
     <string>单曲循环</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="m_onlineButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:checked {
    background-color: lightblue; /* 设置高亮时的背景颜色 */
    border: 2px solid #ADD8E6;
}</string>
    </property>
    <property name="text">
     <string>网络歌曲</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="m_localButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:checked {
    background-color: lightblue; /* 设置高亮时的背景颜色 */
    border: 2px solid #ADD8E6;
}</string>
    </property>
    <property name="text">
     <string>本地歌曲</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QStackedWidget" name="m_stackedWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>361</width>
      <height>321</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="m_pLineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="m_searchButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="m_onlineTableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>371</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>115</number>
      </attribute>
      <column>
       <property name="text">
        <string>歌曲</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>专辑</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>时长</string>
       </property>
      </column>
     </widget>
     <widget class="QTextBrowser" name="m_lyricsBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableWidget" name="m_localTableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>371</width>
        <height>192</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>120</number>
      </attribute>
      <column>
       <property name="text">
        <string>歌曲</string>
       </property>
      </column>
     </widget>
     <widget class="QTextBrowser" name="m_localLyricsBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="m_addMusicButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:checked {
    background-color: lightblue; /* 设置高亮时的背景颜色 */
    border: 2px solid #ADD8E6;
}</string>
    </property>
    <property name="text">
     <string>添加文件</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="kmainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
